### ISSUE TITLE
Add phpstan, phan, psalm and phpmd to project

### ISSUE THREAD

**Issue by koninka** [taptima/phalyfusion#9]

**Comment by kimcheung** [comment]
Hi! Do you want to add these to the prod build?  It looks like it already exists for dev...
https://github.com/taptima/phalyfusion/blob/master/composer.json#L33-L36
if they should be for prod, can I open a PR?
Thanks!

**Comment by koninka** [comment]
@kimcheung 

No, it shouldn't be for prod, because these tools are only needed for development.

The main idea is to check the code of the Phalyfusion by using static analyzers (Phan, Psalm, PHPStan and PHPMD) and later connect the Phalyfusion itself for yourself (#3).

You can set up configs for static analyzers for checking the Phalyfusion code and make a PR for this.
